app/controllers/report_controller.rb

```diff
@@ -0,0 +1,44 @@
+class ReportController < ApplicationController
+  def select_soc_codes
+    report = Report.where(guid: params[:guid]).first
+    @keywords = report.keywords
+  end
+
+  def save_soc_codes
```

Comment:
I see that this is an action method, which feels a bit weird. Can we model this with resources? I haven't really given it tell me if the answer is "lol no"